.github/workflows/ppc64le-image.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and Push Image
```

Comment:
Can you please rename it to something else that is specific for extra arch?

Reply:
yes